src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -192,14 +192,14 @@ const ProjectList = createReactClass({
     return (
       <div className="organization-dashboard-projects">
         <Link className="btn-sidebar-header" to={`/organizations/${org.slug}/teams/`}>
-          View All
+          {t('View All')}
         </Link>
-        <h6 className="nav-header">Projects</h6>
+        <h6 className="nav-header">{t('Projects')}</h6>
         {bookmarkedProjects.length === 0 && (
           <div className="alert alert-info" style={{marginBottom: 10}}>
-            Bookmark your most used{' '}
-            <Link to={`/organizations/${org.slug}/teams/`}>projects</Link> to have them
-            appear here.
+            {tct('Bookmark your most used [projects:projects] to have them appear here', {
```

Comment:
`[projects:projects] ` what is this syntax? does `[projects]` work?

Reply:
Hmm, yeah i think it should be `[projects]`